plugins/backend/catalog-backend-module-okta/README.md

```diff
@@ -19,6 +19,24 @@ catalog:
         token: ${OKTA_TOKEN}
 ```
 
+### OAuth 2.0 Scoped Authentication
+
+[Create an OAuth app](https://developer.okta.com/docs/guides/implement-oauth-for-okta/main/#create-an-oauth-2-0-app-in-okta) in Okta. You will need to grant it with the `okta.groups.read` and `okta.users.read` scopes as a bare minimum. In the following example the `oauth.privateKey` may be passed as either a string encoded PEM or stringified JWK.
+
+```yaml
+catalog:
+  providers:
+    okta:
+      - orgUrl: 'https://tenant.okta.com'
+        oauth: {
```

Comment:
Just a minor comment on format. YAML is a superset of JSON, but any reason for the braces? Just got my attention.

Reply:
Good catch again. Likely just a copy paste blunder